### ISSUE TITLE
#424823 payment conformation email?   no records 

### ISSUE THREAD

**Issue by Irena206** [206Tours/DB#1429]
Hi Brad, 
When you get a chance can you take a look at what here happened ?
So, the client received confirmation that a payment of $3,218 was processed , however we don't have any records that payment wad made/received ?  
Thanks, 

From: orders@206tours.com [mailto:orders@206tours.com] 
Sent: Sunday, August 21, 2016 4:33 AM
To: Binvaldescam@yahoo.com
Subject: 206 Tours Inc. Order #424823: New Payment Received
Importance: High

Dear Virginia,

Thank you for your payment. Attached please find a copy of your updated invoice reflecting today's payment of $3,218.00.

Please feel free to contact us with any questions regarding your payment schedule.

With gratitude,

206 Tours
333 Marcus Blvd
Hauppauge, NY 11788 USA
Tel: (631) 361-4644 or (800) 206-8687
Fax: (631) 361-3682
orders@206tours.com
www.206tours.com

**Comment by bradatkins** [comment]
Is there a record in Elavon for the payment?

**Comment by Irena206** [comment]
No, no record in Elavon, it wouldn't be strange then . Its not ACH request either.
As you can see email was sent out at 4:33AM , so its not like we accidentally approved something in the office.
